### ISSUE TITLE
MDS 2.0 - round-trip car sharing and reservation/reserved

### ISSUE THREAD

**Issue by bergenklem** [CityOfLosAngeles/mobility-data-specification#867]
We experience confusion regarding how the terms "reservation" and "reserved" are being used in MDS 2.0 in relation to use cases of station-based car sharing (A to A), and would like to get some clarification on how to interpret and use the specification!

I've tried to sum up our confusion in the following list:

**1.** The MDS 2.0-documentation states at first that a reservation 
 
>  … is the duration the customer has **_continuous exclusive access_** to the vehicle whether parked or in motion. 

**2** which again may be considered a journey_attribute:
 
> reservation_id (UUID, optional): unique identifier for an entire car share reservation, tied across multiple journeys and therefore trips.
 
The definition of reservation given above is all in line with how we use the term reservation in car sharing operations.

**3** But - when describing vehicle states, «reserved» points towards the state where a car is only reserved by the user (and not put to motion). 

This differs from the definition of «reservation» as mentioned in point 1 above.

**4**. Then it follows up that the transition to/from "reserved" is triggered by the event_types:

> reservation_start
> reservation_stop

**5** At the same time, the trip_type label «reservation» points towards a trip where the car is in motion and reserved by a customer (but not being driven by the customer):

> en route to pickup a customer who has made a reservation, with no passengers in the vehicle
  
So - to sum up:

**A**) We need some clarification on the use of terms "reservation" and "reserved".

One suggestion that could help out, would be to rename event_types "Reservation_start/reservation_stop" to "reserved_start/reserved_stop" to hinder confusion.

**B**) We're struggling a bit to see the use case where a reservation consists of several journeys, especially when the documentation states that a reservation = continuous exclusive access. Right now, it seems like that a reservation is all equal to a journey, and may consist of no more information.

It would make sense if one opens up for concurring reservations (e.g. companies making a reservation for a car every monday (multiple journeys)), but then it still would be available for the other days - and hence not continuously exclusive for the entire duration of reservation.

![Reservation](https://github.com/openmobilityfoundation/mobility-data-specification/assets/134277360/2cf9b1b6-9356-4b4e-9974-1d2d65763bac)

(We've put together a Slides-document to present several of our questions in relation to MDS 2.0 and car sharing, you can find this here: https://docs.google.com/presentation/d/1AbA-gnwvKwZ8BDz7Xp5DzFFRdVNyuiQUHkNIkVn75nY/edit?usp=sharing)

**Comment by schnuerle** [comment]
Thanks for the detailed issue and slides. This would be a good discussion to have during a MDS working group meeting. 

Before that I'd like to get thoughts from others who are in the car share space, like from @jean-populus @jiffyclub at Populus, @jacob-sherman and team in Portland, Ride Report, @StephenRijo in Denver, @alexdemisch SFMTA, etc.

**Comment by schnuerle** [comment]
We will tell the working group about this issue today during the round-robin talk. 

[Slides](https://docs.google.com/presentation/d/13gvfIla7gWjcH-QUL0HOfb2rwVRzHRulRWF2N-UpGIM/edit#slide=id.g247a7dcba93_0_164)

[Meeting agenda](https://github.com/openmobilityfoundation/mobility-data-specification/wiki/Web-conference-notes,-2023.05.25-(MDS-Working-Group))

**Comment by alexdemisch** [comment]
Yes, thank you for the detailed slides.  I agree that this would be a good item for a WG meeting - it'd be good to get a sense of the use cases.

**Comment by marie-x** [comment]
We should update the definition of "reserved" to mean "the time between the user reserving the vehicle and the user taking possession of the vehicle."

**Comment by schnuerle** [comment]
We can clarify this as part of our MDS 2.1 work, which will be kicking off this [week](https://github.com/openmobilityfoundation/mobility-data-specification/wiki/Web-conference-notes,-2024.07.11-(MDS-Working-Group)).

**Comment by schnuerle** [comment]
If we want this to make it to the MDS 2.1 release, we are going to need someone to make a PR in the next month or two so that it can be discussed and reviewed by the steering committee and working group.

**Comment by schnuerle** [comment]
@bergenklem Would you like to attend and present on your ideas here and in your 2 other [open issues around car share](https://github.com/openmobilityfoundation/mobility-data-specification/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22Car%20Share%22%20author%3Abergenklem), at the working group meeting this Thursday?

**Comment by schnuerle** [comment]
From our [WG meeting on Aug 28 2025](https://github.com/openmobilityfoundation/mobility-data-specification/wiki/Web-conference-notes,-2025.08.28-(MDS-Working-Group)):

- No action taken for this one, moved to MDS 3.0 scope
- Haven't been able to demand this data from the operators yet
